### PR TITLE
Podcast Player: Center track icon vertically

### PR DESCRIPTION
**Pull request by WunderBart**
We need an explicit line-height to be sure that the track icon doesn't go off middle in themes that override the default line-height.

Context: https://github.com/Automattic/jetpack/issues/15373#issuecomment-613977242

#### Testing instructions:
* Insert Podcast Player block
* Click on a track to start playback
* Playing icon should be in the middle vertically
* Go offline
* Click one of the episodes to see the error
* Error icon should be in the middle vertically
* All of the above should be true for any theme in both, editor and front-end

#### Screenshots:

| Before  | After |
| ------------- | ------------- |
| Screenshot 2020-04-16 15 05 03|Screenshot 2020-04-16 15 04 12|

#### Proposed changelog entry for your changes:

* none

### CODE REVIEW

**Review by getdave — APPROVED** [review]
I tested on editor and front end and it worked perfectly 🎉

File: extensions/blocks/podcast-player/style.scss [hunk 1]
```diff
@@ -255,6 +255,7 @@ $player-background: transparent;
 			width: $track-status-icon-size;
 			height: $track-status-icon-size;
 			margin-top: ($editor-font-size * $editor-line-height - $track-status-icon-size) / 2; // center vertically
+			fill: inherit;
```
Comment (WunderBart) [inline extensions/blocks/podcast-player/style.scss:258]: This is a bit out of scope. Just noticed that if the fill is not inherited explicitly, some themes (e.g. TwentyNineteen) can override the svg fill.
